### PR TITLE
New radiobutton-based UI for proposal settings

### PR DESCRIPTION
**Pull request by ancorgs**
## Previous dialog

storage-proposal-settings

## New dialog

based

Same UI in ncurses version

based-text

### CODE REVIEW

File: src/modules/StorageProposal.rb [hunk 1]
```diff
@@ -6560,6 +6540,43 @@ def CouldNotDoSeparateHome(prop_target_map)
...
+          strategy_button(:lvm, _("&LVM-based Proposal")),
+          VSpacing(0.3),
+          strategy_button(:lvm_crypt, _("&Encrypted LVM-based Proposal"))
+        )
+      )
+    end
+
+    def strategy_button(strategy, label)
+      selected = self.strategy == strategy
+      Left(RadioButton(Id(strategy), Opt(:notify), label, selected))
```
Comment (aschnell) [inline src/modules/StorageProposal.rb]: Is :notify still needed?

Comment (ancorgs) [inline src/modules/StorageProposal.rb]: Yes, there is a dialog popping up when you select Encrypted LVM.

File: src/modules/StorageProposal.rb [hunk 2]
```diff
@@ -6300,8 +6280,9 @@ def CommonWidgetsHelp()
       # TRANSLATORS: help text
       help_text =
         _(
-          "<p>To create an LVM-based proposal, choose the corresponding button. The\n" +
-          "LVM-based proposal can be encrypted.</p>\n"
+          "<p>Choose Partition-based Proposal if you don't want to use LVM.\n" +
```
Comment (lslezak) [inline src/modules/StorageProposal.rb]: NP: IIRC usually the  button labels are emphasized in the help text so it's obvious it's reference to the UI, e.g. `Choose <b>Partition-based Proposal</b> if ...`

**Comment by lslezak** [comment]
Otherwise LGTM.

**Comment by ancorgs** [comment]
Help text fixed. Waiting for feedback from openQA guys before merging.

**Comment by lslezak** [comment]
Thanks, LGTM from my side...

**Comment by ancorgs** [comment]
Pull request rebased (a new version bump entered master)

**Comment by dzedro** [comment]
It will change/break existing [cryptlvm](https://openqa.suse.de/tests/286092/modules/partitioning_lvm/steps/1) (two) tests, but I can update existing openQA code

**Comment by kobliha** [comment]
@dzedro PLS do so :)

**Comment by ancorgs** [comment]
@dzedro These are the corresponding submit requests, so you can track when they enter the ISOs

https://build.suse.de/request/show/100570
https://build.opensuse.org/request/show/368109

**Comment by dzedro** [comment]
will do so, thx for OBS/IBS links

**Pull request merged**